schemas/rum/_common-schema.json

```diff
@@ -44,6 +44,11 @@
       "description": "Generated unique ID of the application build. Unlike version or build_version this field is not meant to be coming from the user, but rather generated by the tooling for each build.",
       "readOnly": true
     },
+    "ddtags": {
+      "type": "string",
+      "description": "tags of the event",
```

Comment:
blocker/ This description is too vague - it is unclear how to format these tags. Since the type is `string`, it raises questions around separators, key-value structure, and formatting expectations. Including a brief example in the description would help clarify usage and improve consistency across SDKs.